packages/cli/src/cmds/validator/options.ts

```diff
@@ -189,6 +192,14 @@ export const validatorOptions: CliCommandOptions<IValidatorCliArgs> = {
     type: "number",
   },
 
+  disableAttestationGrouping: {
```

Comment:
added this as hidden cli option, I think it does not hurt and it allows us to disable the optimization independent of running in a distributed validator cluster